templates/header.html

```diff
@@ -176,6 +176,15 @@ <h3 class="warning">Information</h3>
               </div>
             </li>
 
+            {# admin #}
+            {% if user.is_superuser %}
+            <li class="nav-item">
+              <div class="nav-link yt-main-link">
+                <a href="{% url 'admin:index' %}" title="Admin"><b><i class="fas fa-cog"></i>&nbsp;Admin</b></a>
+              </div>
+            </li>
+            {% endif %}
```

Comment:
This section I feel is superfluous. We don't use the django login for login to YATA, it would be more benificial to print the admin url to console/log on running the application and advise the user what this is I feel.

Reply:
Hey Jack, notice the conditional statement before the link, this ensures that only superuser will see it. The reason I added a link is because it will be easier to navigate to as a super user. I presumed that there are super users using that login right?

Reply:
I did yes, I mean I guess it can stay, it's just a bit superfluous as I say :) Will definitely need something in the console log though on start else people won't know what their admin URL is!

Reply:
Admin login / yata login are totally different user systems, you can't don't login to the player side of yata as a super admin.

Let's imagine it's my first ever run of YATA, I don't even know what my admin URL is to login as a super user to begin with

Reply:
Yes, I totally agree that yata user != django use. A Yata user would never see this Admin cog nav link, nor would they ever have to use it. This admin link only displays when a valid django super user has authenticated. Happy to remove this link though if you want me to and add a console log. My only worry about that is you won't be able to get to it on the production site unless you login to the server and check the logs :O

Reply:
Removed the link